standard/array.lua

```diff
@@ -263,13 +308,19 @@ Example:
 Array.append({2, 3}, 5, 7, 11)
 -- returns {2, 3, 5, 7, 11}
 ]]
+---@generic T
+---@param tbl T[]
+---@param ... any
+---@return T[]
```

Comment:
Since the any is added to the T, this is not correct

Reply:
hope the last commit is what you want^^

Reply:
Not exactly, but good enough for now